Frames.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Frames"
-  s.version      = "3.5.2"
+  s.version      = "3.5.3"
```

Comment:
@harry-brown-cko For 3.5.3 release : The tag was created in develop never merged into master, we have to update master.